sdk/cognitivelanguage/ci.yml

```diff
@@ -28,3 +28,5 @@ extends:
     Artifacts:
     - name: Azure.AI.Language.Conversations
       safeName: AzureAILanguageConversations
+    - name: Azure.AI.Language.QuestionAnswering
+      safeName: AzureAILanguageQuestionAnswering
```

Comment:
You didn't need to update any test assertions? Now I wonder why I did in Python...

Reply:
Didn't you sign off on my PR yesterday that had a whole bunch of test updates. ;)